modules/admin_manual/pages/release_notes.adoc

```diff
@@ -30,68 +30,87 @@
+With this new feature, users can control the lifetime of shares with other users or groups. 
+Administrators can choose to set a default maximum lifetime and to enforce it. 
+To integrate this change, the UI in the user/group sharing tab of the sidebar has been adapted. 
+When a resource is shared, the user and group entries are expandable and collapsible using the cogwheel next to the trash bin icon to show/hide the permissions and the expiration date field to maintain an overview. 
+Additionally, to allow users to recognize expiring shares at a glance, a new clock indicator will be shown next to the cog wheel.
 
-Administrators can configure the feature in the 'Sharing' section of the admin settings.
+Administrators can configure the feature in the '_Sharing_' section of the admin settings.
 
-=== Sharing information in subfolders
+=== Sharing Information in Subfolders
 
-ownCloud Server 10.4 puts focus on user awareness for shared areas to prevent accidentally sharing data or changing other users' data as well as to make it easier for users to recognize who has access to shared areas. Practically, users can now better recognize shared resources using a new share overlay indicator on file and folder icons. The indicators are also applied to resources which are not directly shared but are part of a share (when working in a shared folder).
+ownCloud Server 10.4 puts the focus on user awareness for shared areas to prevent accidentally sharing data or changing other users' data, as well as to make it easier for users to recognize who has access to shared areas. 
+Practically, users are better able to recognize shared resources using a new share overlay indicator on file and folder icons. 
+The indicators are also applied to resources that are not directly shared but are part of a share (when working in a shared folder).
 
-Apart from that, the sharing sidebar panels have been improved to also show users/groups and public links that have access through shares on parent folders. These will be shown as static entries with a "via" indicator that allows users to jump to the parent folder and to change the share properties if desired.
+Apart from that, the sharing sidebar panels have been improved to also show users/groups and public links accessible through shares on parent folders. 
+These will be shown as static entries with a "_via_" indicator that allows users to jump to the parent folder and to change the share properties, if desired.
 
-This sharing information is only shown to share owners (users that created shares) as other share recipients are not entitled to get detailed information about who else has access.
+This sharing information is only shown to shareowners (users that create shares) as other share recipients are not entitled to get detailed information about who else has access.
```

Comment:
- Really "shareowners"? Looks strange to me :)
- I think "created" wasn't that bad as the shares need to exist for that the mentioned sharing information is shown.

Reply:
shareowners is another valid spelling.